### PR TITLE
Add to shipment from master list (f/e)

### LINKED ISSUE #117: Outbound: add from master list
Coming from these issues: 

- https://github.com/openmsupply/remote-server/issues/999
- https://github.com/openmsupply/openmsupply-client/issues/1015

and as requested by a potential client. 
Can we add the functionality to "add from master list" when editing an outbound shipment?
Mobile has this, though mSupply does not.

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #117 

Finishes the changes - this part is the front end implementation

## Testing
Note: the b/e PR checks most of the functionality.
- [ ] There should be a button in the app bar
- [ ] It will be enabled when the add item button is
- [ ] You can add items from a master list

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/distribution.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
   "messages.click-to-return-to-requisitions": "Unable to find a requisition with that ID. Click OK to return to the requisition list",
   "messages.click-to-return-to-shipments": "Unable to find a shipment with that ID. Click OK to return to the shipment list",
   "messages.confirm-add-from-master-list": "Do you want to add all of the items from this master list?",
-  "messages.confirm-status-as": "Confirm status as {{status}}?",
+  "messages.confirm-status-as": "Confirm status as $t({{status}})?",
```
Comment (mark-prins) [inline client/packages/common/src/intl/locales/en/distribution.json:91]: oops! I had 'corrected' these in another PR and I now see that they are supposed to be like this 🙄

File: server/configuration/base.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ server:
       https://demo-open.msupply.org,
       http://localhost:8000,
     ] # Used to set the allowed Origin in Cross Origin Requst Security
+  base_dir: "app_data"
```
Comment (mark-prins) [inline server/configuration/base.yaml:10]: had to add this in order to get cargo to run after the merge from main

**Comment by mark-prins** [comment]
Note: After merging main I've had to add `  base_dir: "app_data"` to my local.yaml in order to get cargo to run.

**Review by clemens-msupply — APPROVED** [review]
Works and looks all good to me!

**Pull request merged**